### ISSUE TITLE
2021S Application of YOLO v2 Object Detection(Group work)

### ISSUE THREAD

**Issue by Jin1129** [Jin1129/Assignments_Jin_uts#1]
Object detection refers to using the theory and method in the fields of image processing and pattern recognition to detect target objects in an image. It can determine the category of these target objects and mark the position of the target object in the image. Object detection is the prerequisite for object recognition. Only when the object is detected can the object be recognized.

The input of object detection is an image or video sequence frame, and the underlying features of the image can be described by a feature extraction algorithm. Select representative features from feature vectors and reduce the dimensionality of features. Then, a specific classifier is used to classify the features and determine whether the candidate area contains the target and its category. Finally, the intersecting candidate areas of the same category are judged, and the bounding box of each target is calculated to complete the object detection.

Object detection has a wide range of applications in the fields of face recognition, medical imaging, intelligent video surveillance, robot navigation, image editing, and augmented reality. Using theories and methods in the fields of image processing and pattern recognition, we can separate meaningful entities such as people, cars, buildings and other objects from images or videos.

In this project, the dataset we selected is a video sequence frame on a highway (refer to Figure 1). The data set includes a training set and a test set. The training set has 1000 sequence frames, and the test set has 176 sequence frames. It can detect whether there is a vehicle in the image and mark the detected vehicle.
